vowpalwabbit/parser/flatbuffer/schema/example.fbs

```diff
@@ -10,6 +10,7 @@ table Namespace {
   name:string;
   hash:uint8;
   features:[Feature];
+  full_hash:uint64;
```

Comment:
a /// comment for this element would be useful.

Reply:
Will do. I'd rather call this hash but that is already taken by the other element which is really the index.

Reply:
Yeah.  I agree with your preference.

Reply:
Should I switch them? I am worried that will break things. What do you think?

Reply:
Going to leave as is with a comment